src/int.cr

```diff
@@ -155,14 +155,18 @@ struct Int
   #
   # See `Int#/` for more details.
   def %(other : Int)
-    if other == 0
-      raise DivisionByZeroError.new
-    elsif (self < 0) == (other < 0)
-      self.unsafe_mod(other)
-    else
-      me = self.unsafe_mod(other)
-      me == 0 ? me : me + other
-    end
+    {% begin %}
+      if other == 0
+        raise DivisionByZeroError.new
+      elsif self < 0 && self == {{@type}}::MIN && other == -1
+        self.class.new(0)
+      elsif (self < 0) == (other < 0)
+        self.unsafe_mod(other)
+      else
+        me = self.unsafe_mod(other)
```

Comment:
What does `me` stand for? Could you maybe while you are on it write this variable out to make the meaning more clear?

Reply:
It's a good idea but it can be done in a separate PR to avoid mixing intentions.

Reply:
Okay, sure. Should I open an issue or will you make the PR? I would do it but I don't know what `me` stands for and can't find it anywhere else.

Reply:
It's just two lines. It's the result of a function.

Reply:
I'll maybe send a PR later, I don't think it's a big deal. It's just a temporary result. The code is easy to understand looking at just those two lines.